code/Language/Drasil/Sentence/Math.hs

```diff
@@ -0,0 +1,8 @@
+module Language.Drasil.Sentence.Math where
```

Comment:
I would prefer to call this `Language.Drasil.Sentence.EmbedSymbol`

Reply:
Change were made in 9bc2e7f6da99a6717fd96c5817fbf2d62e1903f4.